machine-learning/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.11 as builder
+FROM python:3.10 as builder
```

Comment:
3.10?

Reply:
Ray has some issues with 3.11 at the moment ([relevant issue](https://github.com/ray-project/ray/issues/36319)). I'll switch back to 3.11 once it's working.

Reply:
I tried to make the [AIO image](https://github.com/imagegenius/docker-immich) compatible with your amazing work, but couldn't find a way (tried to build python 3.10 from source and to use deadsnake PPA). I suppose we'll disable machine-learning until ray officially supports it

Reply:
Hmm, what kind of error are you getting? It should work if you're using 3.10. Could it be a missing dependency?

Reply:
Yep, but don't worry, I found a work around with a builder. Not ideal, but it works

Reply:
It looks like Ray will be releasing a fix for 3.11 soon. Once it's out I think we can merge without downgrading Python (assuming there are no other issues).